src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/TravellerSelection.tsx

```diff
@@ -35,6 +35,7 @@ type TravellerSelectionProps = {
   style?: StyleProp<ViewStyle>;
   selectionMode: TravellerSelectionMode;
   fareProductType: string;
+  fareProductOnBehalfOfEnabled: boolean;
```

Comment:
It may be time to just send in the `fareProductTypeConfig` object instead of three separate fields here.

Reply:
@gorandalum the fareProductType in here originates from `PreassignedFareProduct`, not sure if it is safe to switch to `FareProductTypeConfig`, I'll need to check if its safe to switch, if it is safe, then I will pass the `fareProductTypeConfig`

Reply:
They should match, or else something is really wrong 😅

Reply:
Updated 🕺